src/components/Drawer.vue

```diff
@@ -168,15 +168,20 @@ export default class Drawer extends Vue {
 
   created() {
     if (this.phoneLayout) {
-      return
+      this.endItems = this.endItems.concat([
+        { icon: 'mdi-card-search-outline', title: 'Search', click: () => this.updateOptions('showSearch', true) },
+      ]);
+
+      return;
     }
 
     this.endItems = this.endItems.concat([
       { icon: 'mdi-rss-box', title: 'RSS', click: () => this.updateOptions('showRss', true) },
+      { icon: 'mdi-card-search-outline', title: 'Search', click: () => this.updateOptions('showSearch', true) },
```

Comment:
Store this menu item in variable.

Reply:
Well, I will do a refactoring on using cards on the phone instead of the data-table. To be honest it is a pretty good feature. 

For example, Two weeks ago I was in the park with friends and one of them told me about a movie I must watch. I open the UI from my phone and download it and when I got back I already had it downloaded and just started watching it. Cool huh?